courses/free-courses-ar.md

```diff
@@ -7,6 +7,8 @@
 * [C](#c)
 * [C Sharp](#c-sharp)
 * [Clean Code](#clean-code)
```

Comment:
Seen [subject lists](https://github.com/EbookFoundation/free-programming-books/blob/master/books/free-programming-books-subjects.md#software-architecture) I think is better join all this branch (your additions included) into same category and renaming to `Software Architecture`

Reply:
Great !
But do the repo rules allow me to use a 3rd level header as a subtitle?
As in this picture 
![Screenshot from 2021-11-01 00-40-00](https://user-images.githubusercontent.com/79117370/139603806-699e6e00-ca60-4559-bdf5-38cc78fe6443.png)

Reply:
Did you mean to rename "Clean Code"  to "Software Architecture"?

Reply:
Do you want me to write them as
- Software Architecture
  - Clean Code
  - Design Patterns
  - Solid

Reply:
Plain. Only `Software Architecture`. Move all resources there